specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -9588,6 +9791,14 @@
       "minimum": 0,
       "maximum": 500,
       "x-ms-parameter-location": "method"
-    }
+    },
+    "ModelIdInPath": {
```

Comment:
I don't want to use more parameters in our Swagger file. They don't get generated automatically from the controllers. So replace the ModelInPath references with the definition in-line.